content/en/apps/guides/performance/telemetry.md

```diff
@@ -97,4 +97,22 @@ When the aggregate doc is created the Telemetry service also includes a snapshot
 | `dbInfo.idb_attachment_format` | The format of database attachments. |
 | `dbInfo.db_name` | The name of the local database. |
 | `dbInfo.auto_compaction` | Whether or not auto compaction is set. |
-| `dbInfo.adapter` | The database adapter being used. |
\ No newline at end of file
+| `dbInfo.adapter` | The database adapter being used. |
+
+## Export data to JSON
+
+{{% alert title="Note" %}}
+Telemetry data can be viewed directly in your browser with [Fauxton](https://couchdb.apache.org/fauxton-visual-guide/) at `https://{{CHT_INSTANCE_URL}}/_utils`, and navigating the `medic-users-meta` database.
+{{% /alert %}}
```

Comment:
Do you need an empty line after the close of the alert to display well in Hugo?

Reply:
I checked and it works any which way. As it is now:

```
and navigating the `medic-users-meta` database. 
{{% /alert %}}
Follow the steps below to export all telemetry in JSON for further analysis or visualization:
```

With an extra line:

```
and navigating the `medic-users-meta` database. 
{{% /alert %}}

Follow the steps below to export all telemetry in JSON for further analysis or visualization:
```

With no line extra lines:

```
and navigating the `medic-users-meta` database. {{% /alert %}}Follow the steps below to export all telemetry in JSON for further analysis or visualization:
```

Unless there's a best practice I'm missing, I'll keep it as is!

Reply:
Actually...an extra line after the `{{% /alert %}}` does indeed make it more legible in code ;)